### PR TITLE
Loqus version req

### PR DESCRIPTION
**Pull request by moonso**
This PR adds a requirement on loqusdb to use `version >= 2.5`

**How to test**:
1. Install on local machine 
1. Create a loqusdb environment with a loqusdb version <= 2.5
1. Start the server

**Expected outcome**:
Scout should exit gracefully and tell what was the problem
Skärmavbild 2020-03-19 kl  10 15 37

**Review:**
- [ ] code approved by
- [x] tests executed by @moonso

### CODE REVIEW

File: docs/admin-guide/loqusdb_integration.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Loqusdb integration
 
-[Loqusdb][loqus] is a tool to keep track of local observation of genomic variants. Local frequencies/observations has been shown to add value to the analysis of genomic data both for rare disease and cancer. Scout integrates with loqusdb by fetching data from a given instance of loqusdb via the CLI.
+[Loqusdb][loqus] is a tool to keep track of local observation of genomic variants. Local frequencies/observations has been shown to add value to the analysis of genomic data both for rare disease and cancer. Scout integrates with loqusdb by fetching data from a given instance of loqusdb via the CLI. Please note that loqusdb version >= 2.5 needs to be installed.
```
Comment (northwestwitch) [inline docs/admin-guide/loqusdb_integration.md]: Perhaps write that if you are using a Scout version >= n then you need loqusdb version >= 2.5

File: scout/commands/base.py [hunk 2]
```diff
@@ -56,22 +56,30 @@ def get_app(ctx):
...
-            or cli_config.get("host")
-            or "localhost",
-            MONGO_PORT=options.params.get("port") or cli_config.get("port") or 27017,
-            MONGO_USERNAME=options.params.get("username") or cli_config.get("username"),
-            MONGO_PASSWORD=options.params.get("password") or cli_config.get("password"),
-            OMIM_API_KEY=cli_config.get("omim_api_key"),
-        ),
-        config_file=flask_conf,
-    )
+    try:
```
Comment (northwestwitch) [inline scout/commands/base.py]: 👍

File: scout/server/extensions/loqus_extension.py [hunk 3]
```diff
@@ -70,6 +70,13 @@ def init_app(self, app):
         self.version = app.config["LOQUSDB_SETTINGS"].get("version")
         if not self.version:
             self.version = self.get_version()
+        self.version_check()
+
+    def version_check(self):
+        """Check if a compatible version is used"""
+        if not self.version >= 2.5:
+            LOG.info("Please update your loqusdb version to >=2.5")
+            raise SyntaxError("Only compatible with loausdb version >= 2.5")
```
Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py]:
```suggestion
            raise SyntaxError("Only compatible with loqusdb version >= 2.5")
```

File: tests/server/extensions/test_loqusdb_extension_init.py [hunk 4]
```diff
@@ -90,6 +91,19 @@ def test_init_loqusextension_init_app_no_version(mocker, loqus_exe, loqus_versio
         assert loqus_obj.loqusdb_config is None
 
 
+def test_init_loqusextension_init_app_wrong_version(loqus_exe):
```
Comment (northwestwitch) [inline tests/server/extensions/test_loqusdb_extension_init.py]: 👍

File: docs/admin-guide/loqusdb_integration.md (hunk 1 above)
Comment (moonso) [reply docs/admin-guide/loqusdb_integration.md]: No it will never work with loqus < 2.5 🤕

**Comment by moonso** [comment]
Thanks @northwestwitch !

**Pull request merged**